app/views/activities/_download_as_xml.haml

```diff
@@ -0,0 +1,4 @@
+- if policy(activity).download?
+  .govuk-grid-row
+    .govuk-grid-column-full
+      = link_to t("default.button.download_as_xml"), organisation_activity_path(activity.organisation, activity, format: :xml), class: "govuk-button"
```

Comment:
We've had this as a "tech debt" for ages, but it never came up as a need from BEIS users 🤷🏻 It's rarely used anyway, since the most XML downloads happen from the Exports page.